man/gbz80.7

```diff
@@ -1833,7 +1874,19 @@ Flags: See
 .Sx SRL r8
 .Ss STOP
 Enter CPU very low power mode.
-Also used to switch between double and normal speed CPU modes in GBC.
+Also used to switch between GBC double speed and normal speed CPU modes.
+.Pp
+The exact behavior of this instruction is fragile and may interpret its second byte as a separate instruction
+.Po see
+.Lk https://gbdev.io/pandocs/Reducing_Power_Consumption.html#using-the-stop-instruction the Pan Docs
+.Pc ,
+which is why
+.Xr rgbasm 1
+outputs a
+.Sy NOP
+byte as the second
+.Sy STOP
+byte by default if one is not specified.
```

Comment:
Perhaps an example should be given of how to do so? Otherwise, it's not clear that the syntax is `stop $AF`.

Reply:
The opcode table documents it as "`STOP n8`"; should we do the same here to make that clear?

Reply:
I think we should keep the canonical definition as just `STOP`, and mention the optional `STOP n8` syntax in the description, since it's really uncommon.

Reply:
```mdoc
which is why
.Xr rgbasm 1
allows explicitly specifying the second byte
.Pq Sy STOP Ar n8
to override the default of $00.
```

Reply:
Please turn `to override the default of $00.` into `to override the default of $00 (NOP).` and I'm fine with the latest suggestions.